azurerm/internal/services/compute/tests/data_source_dedicated_host_test.go

```diff
@@ -0,0 +1,40 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceAzureRMDedicatedHost_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_dedicated_host", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { acceptance.PreCheck(t) },
+		Providers: acceptance.SupportedProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceDedicatedHost_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(data.ResourceName, "id"),
```

Comment:
checking the ID is superflurious (since this'll fail without it) - can we check the specific fields here?